tests/utils/sampler_test.py

```diff
@@ -177,7 +177,7 @@ def test_dirichlet_sampler_without_hash_key() -> None:
 
     _, p_val_1 = chisquare(f_obs=new_samples_per_class_1, f_exp=samples_per_class)
     _, p_val_2 = chisquare(f_obs=new_samples_per_class_2, f_exp=samples_per_class)
-    _, p_val_3 = chisquare(f_obs=new_samples_per_class_1, f_exp=new_samples_per_class_2)
+    _, p_val_3 = chisquare(f_obs=new_samples_per_class_2, f_exp=new_samples_per_class_1)
```

Comment:
I noticed a div by zero warning was popping out from this test. It's caused by the fact that the `new_samples_per_class_2` distribution has one class with 0 elements. `new_samples_per_class_1` doesn't have this issue. So making it the reference distribution and the other the test distribution eliminates the error and should be effectively the same statistical test.